modules/arrow/package.json

```diff
@@ -35,9 +35,9 @@
     "build-worker": "webpack --entry ./src/workers/arrow-worker.js --output ./dist/arrow-worker.js --config ../../scripts/webpack/worker.js"
   },
   "dependencies": {
+    "@apache-arrow/es5-cjs": "^4.0.0",
```

Comment:
I'm guessing we can't use `@apache-arrow/ts` because we don't have a `tsc` build step in loaders right now? Otherwise seems it might help with the tsconfig issues? It would be great if the loaders build process were improved so that modules could optionally write in `.ts` files directly, but not sure how hard that would be.

Reply:
Yes, and loaders currently uses `d.ts` files for typing. Additionally, as my earlier comment says, seems to be a typing problem on the apache arrow side itself? I switched to using `@apache-arrow/es5-cjs` thinking it would omit the types (until that issue is fixed), but it looks like they ship them with it anyways.